src/test-apps/happy/bin/weave-register-service.py

```diff
@@ -29,7 +29,6 @@
 import sys
 import set_test_path
 
-from happy.Utils import *
```

Comment:
If we are removing this file from having dependencies on Happy, then it needs to be moved out of src/test-apps/happy/bin/... and then needs to be moved somewhere else.

Reply:
@gerickson  -- I agree in principle.  In the recent few weeks we've had a number of conversations around taking the service account manager and landing it in a more sane place.  The list of work items on that front included:

- refactoring the class to make it more independent from GRPC classes loaded from internet
- folding the class into some UI aligned with the weave device manager
- amending this code to work with other authentication schemes that are on the horizon for Google Nest Weave service.  

This set of changes was created opportunistically -- a third party collaborator asked to be able to run this script on MacOS.   The dependencies this script had on happy were superficial, and @akrithika27 stepped up to solve them.  

In practical terms: 
* we do need a design document the chart the direction for service account manager changes going forward. This is a substantial undertaking perhaps outside the scope of this surgical change.  I will take an action to drive the design doc, and we will review it shortly.
* The broader design will propose a better location for this script. Right now, a better place to locate these scripts would be either in `src/device-manager/python` (this begs the larger redesign decision) or in `src/test-apps/`; perhaps creating something like `src/test-apps/bin/` or `src/test-apps/scripts` .  If moving the scripts to either of the latter three locations makes the change more acceptable, I would completely agree and support that request. 
* I want to enable the third party collaborator without blocking on the larger changes.

In light of the above reasoning, I would suggest we approve the changes as they are (with perhaps moving the scripts to one of the locations listed above.